helpers/lib/cdnify.js

```diff
@@ -60,7 +60,7 @@ module.exports = globals => {
             return path;
         }
 
-        if (path[0] !== '/') {
+        if ((path !== 'img/icon-sprite.svg') && (path[0] !== '/')) {
```

Comment:
I wouldn't want to introduce any hardcoded paths like this. Let's dig deeper to find the underlying root cause.

Reply:
@mattolson I pushed a change for that